app/models/visit.rb

```diff
@@ -18,7 +18,10 @@ class Visit
   }
   validate :validate_amount_of_adults, on: :visitors_set
 
-  delegate :prison_name, :prison_name=, to: :prisoner, allow_nil: true
+  delegate :prison, :prison_name, :prison_name=,
+    to: :prisoner, allow_nil: true
+  delegate :prison_email, :prison_canned_responses, :prison_nomis_id,
+    to: :prisoner
 
   def validate_amount_of_adults
     if visitors.none? { |v| v.age && v.age >= 18 }
```

Comment:
@dan-palmer you must be right. It's even starker [in context](https://github.com/ministryofjustice/prison-visits/blob/0e2a556899bc368e1abefbb7c28325f59d665404/app/models/visit.rb#L27-L32):

```ruby
    if visitors.none? { |v| v.age && v.age >= 18 }
      errors.add :visitors, :at_least_one_adult
    end
    if visitors.count { |v| v.age && v.age >= adult_age } > 3
      errors.add :visitors, :max_3_adults, adult_age: adult_age
    end
```

Those can't both be correct, surely?